### PR TITLE
added shadows to cards [RFR]

### PR DESCRIPTION
**Pull request by mandym-webdev**
Added drop shadows as per https://www.pivotaltracker.com/story/show/97115062

### CODE REVIEW

File: client/src/index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   <meta property="og:description" content="¡Conoce la ubicación del patrimonio de Medellín!" />
   <meta property="fb:app_id" content="1124058024274759" />
   <meta property="og:image" content="../images/facebook-image.png" />
-
```
Comment (lifehackett) [inline client/src/index.html]: This change is in your fb/twitter branch I believe (and seems more related to those changes)

File: client/src/styles/leafletComponents/filters.less [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .c-filters {
-
```
Comment (lifehackett) [inline client/src/styles/leafletComponents/filters.less:2]: Phew! This was the missing touch :smile:

File: client/src/styles/leafletComponents/controlFrame.less [hunk 3]
```diff
@@ -42,9 +43,13 @@
...
     margin-top: 10px;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
   }
 
   .c-control-frame__results {
     .s-container;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
+
   }
```
Comment (lifehackett) [inline client/src/styles/leafletComponents/controlFrame.less]: Yuck!  So a couple things here
1) This is not very DRY.  If you instead add it to `.s-container` which they all reference then you will get the same effect but only have to write it once.  You may need to add `.s-container` to the currently empty `.c-control-frame__search`
2) There is a LESS mixin for box-shadow that would be good to use.  http://lesselements.com/

File: client/src/styles/leafletComponents/search.less [hunk 4]
```diff
@@ -5,6 +5,7 @@
     font-weight : 400;
     padding-left: 5px;
     border      : none;
+    box-shadow: 0 3px 6px rgba(0,0,0,0.16), 0 3px 6px rgba(0,0,0,0.23);
```
Comment (lifehackett) [inline client/src/styles/leafletComponents/search.less:8]: See comments above

Comment (mandym-webdev) [inline client/src/styles/leafletComponents/search.less:8]: ok, i will work on this.

**Comment by mandym-webdev** [comment]
@lifehackett made the changes to dry it up.  Didn't do the mixin because I would have needed to add the elements.less file, call it, etc. which seemed a bit of overkill since we were only using one element which can be covered in one line of code.  I can add it though if you want.  There are conflicts so I will do the merge and deal with them once you've given me the ok.

**Pull request merged**